core/src/main/scala/org/apache/spark/api/r/RRunner.scala

```diff
@@ -124,7 +125,8 @@ private[spark] class RRunner[U](
       partitionIndex: Int): Unit = {
     val env = SparkEnv.get
     val taskContext = TaskContext.get()
-    val bufferSize = System.getProperty("spark.buffer.size", "65536").toInt
+    val bufferSize = System.getProperty(BUFFER_SIZE.key,
+      BUFFER_SIZE.defaultValue.get.toString).toInt
```

Comment:
`BUFFER_SIZE.defaultValueString` should work?

Reply:
done